mobify/platform-scaffold
# [Base] SFCC Connector move

This PR contains the work for the SFCC connector move. All the work has been reviewed in separate PRs. 

 **JIRA**: https://mobify.atlassian.net/browse/WEBDATA-22
 **Linked PRs**: n/a

## To Do
- ~Add Checkout branch of the store to the SDK https://github.com/mobify/progressive-web-sdk/pull/726 & #831~
- ~Add form selectors to the SDK https://github.com/mobify/progressive-web-sdk/pull/726 & #831~

## How to test-drive this PR
- run `npm i`
- run `npm run dev`
- preview the SFCC site
- navigate through the site
- everything should work as expected
